src/libcharon/plugins/kernel_super/kernel_super_plugin.c

```diff
@@ -218,6 +274,12 @@ plugin_t *kernel_super_plugin_create(void)
 		return NULL;
 	}
 
+	if (! init_system_events())
```

Comment:
Continuing with the above comment, remove this function (the system_events instance should be stored in a data member of `kernel_super_ipsec_t`), and move `init_system_events()` to kernel_super_ipsec.c (see routes_conn as a reference)